.php_cs

```diff
@@ -1,29 +1,22 @@
 #!/usr/bin/env php
 <?php
 
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
     ->exclude('var')
     ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
```

Comment:
I would keep excluding them (no need to look for them to discover they are unsupported, while we can skip entering these folders entirely)